src/main/java/codeu/model/data/User.java

```diff
@@ -35,7 +35,7 @@
   public User(UUID id, String name, String password, Instant creation) {
```

Comment:
You forgot to change the 'password' parameter to 'hashedPassword'

Reply:
Got it